universal-application-tool-0.0.1/app/services/program/predicate/PredicateValue.java

```diff
@@ -46,16 +45,21 @@ public static PredicateValue of(ImmutableList<String> value) {
             .map(PredicateValue::surroundWithQuotes)
             .collect(toImmutableList())
             .toString(),
-        ScalarType.LIST_OF_STRINGS);
+        OperatorRightHandType.LIST_OF_STRINGS);
+  }
+
+  public static PredicateValue ofListOfLongs(ImmutableList<Long> value) {
+    return create(
+        value.stream().collect(toImmutableList()).toString(), OperatorRightHandType.LIST_OF_LONGS);
```

Comment:
Does this work?

```suggestion
        value.toString(), OperatorRightHandType.LIST_OF_LONGS);
```

Reply:
Oh! I guess maybe I misunderstood what needed to be donee here. Seems like this should work. :)